src/Entity/EntityId.php

```diff
@@ -17,6 +18,30 @@
 
 	protected $serialization;
 
+	const PATTERN = '/^:?(\w+:)*[^:]+\z/';
```

Comment:
@thiemowmde I think you're looking at an outdated version of ItemId and PropertyId. Github shows some strange things on the "Conversation" page of the pull request since they introduced their new review features...